crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -54,6 +56,12 @@ pub const SKINNING_HANDLE: HandleUntyped =
 
 impl Plugin for MeshRenderPlugin {
     fn build(&self, app: &mut bevy_app::App) {
+        load_internal_asset!(
+            app,
+            MESH_VERTEX_OUTPUT,
+            "mesh_vertex_output.wgsl",
+            Shader::from_wgsl
+        );
```

Comment:
Why was this explicitly split out in this PR? Seems like a good idea, but the reasoning is unclear.

Reply:
because of 
> fragment inputs should now exactly match vertex outputs for locations, so I added exports for those to be able to reuse them https://github.com/gfx-rs/wgpu/pull/2704

a few shaders from examples didn't use the exact same fragment input as the vertex output which caused issues. by having an imported shader for it I can guarantee it's the same